tenable_io/client.py

```diff
@@ -93,6 +93,12 @@ def wrapper(*args, **kwargs):
                     return f(*args, **kwargs)
                 except TenableIORetryableApiException as exception:
                     count += 1
+                    response = exception.response
```

Comment:
We should keep all the error logging in the `_request` method, use `logging.error` when `response.status_code` is not `2XX` in additional to the `logging.debug`.